crates/provider/src/ext/debug.rs

```diff
@@ -128,6 +128,23 @@ pub trait DebugApi<N, T>: Send + Sync {
         state_context: StateContext,
         trace_options: GethDebugTracingCallOptions,
     ) -> TransportResult<Vec<GethTrace>>;
+
+    /// The `debug_executionWitness` method allows for re-execution of a block with the purpose of
+    /// generating an execution witness. The witness comprises of a map of all hashed trie nodes to
+    /// their preimages that were required during the execution of the block, including during
+    /// state root recomputation.
+    ///
+    /// The first argument is the block number or block hash. The second argument is a boolean
+    /// indicating whether to include the preimages of keys in the response.
+    ///
+    /// # Note
+    ///
+    /// Not all nodes support this call.
+    async fn debug_execution_witness(
+        &self,
+        block: BlockNumberOrTag,
+        include_preimages: bool,
```

Comment:
we removed the additional bool

https://github.com/paradigmxyz/reth/blob/870ffae9094e411343781f7a1dc29d503740f168/crates/rpc/rpc-api/src/debug.rs#L142-L142

Reply:
whoops, removed, looks like we need to update the reth doc comment as well

Reply:
https://github.com/paradigmxyz/reth/pull/12559